templates/test/details.html.ep

```diff
@@ -21,6 +21,7 @@
                     <td class="component">
                         <div>
                             %= link_to $module->{name} => url_for('src_step', stepid => 1, moduleid => $module->{name}, testid => $testid)
+                            <button onclick="showThumbnails('<%== $i %>')">+/-</button>
```

Comment:
I guess this can look a bit more pretty. At least the '/-' does not make sense for me and could be simply replaced by just '+'. Even better some icon from http://fontawesome.io/icons/, e.g.
[expand](http://fontawesome.io/icon/expand/), [plus](http://fontawesome.io/icon/plus/), [plus-square-o](http://fontawesome.io/icon/plus-square-o/), [search-plus](http://fontawesome.io/icon/search-plus/)